### DISCUSSION TITLE
Selecting databases on other clusters

### DISCUSSION THREAD

**Discussion by ItsRobbAllen** [ravendb/ravendb#15081] [Ideas]
<img width="691" alt="image" src="https://user-images.githubusercontent.com/32247592/194616021-a5bd7aae-40b3-40a4-bb3f-8c538133bb61.png">

Is there a specific reason why I cannot switch databases from this dropdown in Raven Studio (Cloud) when the database is on another cluster? I have to click on Databases & select it there, which seems like extra work.

**Comment by ayende** [comment]
Nitpick, it isn't on a different cluster, it is on a different *node* in the same cluster.

The reason is that we'll need to open a new tab for that, and in the switcher you don't really expect that.

**Reply by ItsRobbAllen** [reply]
Apologies for the delay AND calling a node a cluster ;)

Thanks. I get it's a design decision.
